src/app/compressed-air-assessment/results-panel/inventory-performance-profile/inventory-performance-profile.component.ts

```diff
@@ -235,6 +240,40 @@ export class InventoryPerformanceProfileComponent implements OnInit {
         }
       });
 
+      if(avgOpPointData){
+        avgOpPointData.forEach(dataItem => {
+          let currentTraceColor: string;
+          if (this.showAllCompressors) {
+            currentTraceColor = this.plotlyTraceColors[currentColorIndex];
+            if (currentColorIndex == 9) {
+              currentColorIndex = 0;
+            } else {
+              currentColorIndex++;
+            }
+          }
+          let unloadingControlTypes: Array<number> = [2, 3, 4, 5, 8, 10];
```

Comment:
Change to use this.unloadingControlTypes and please change the one above as well